README.md

```diff
@@ -113,6 +113,25 @@ App::make('Foo')->bar();
 
 For more information on what features are available on the `Swis\Laravel\Mautic\Client` class, check out the Mautic docs at [https://developer.mautic.org/#endpoints](https://developer.mautic.org/#endpoints), and the manager class at [https://github.com/GrahamCampbell/Laravel-Manager#usage](https://github.com/GrahamCampbell/Laravel-Manager#usage).
 
+# Notifications
+
+To use the ```NotificationDriver``` built into this package make sure the entity you want to notify has the following traits:
```

Comment:
The formatting suggests `NotificationDriver` is an actual class. I'd just write is as 'notification driver' (without the quotes).